### PR TITLE
app: removing Talk

### PR DESCRIPTION
**Pull request by arthyn**
PR Checklist
- [X] Includes changes to desk files
- [X] Describes how you tested the PR locally (test ship vs livenet)
- [ ] If a new feature, includes automated tests
- [ ] Comments added anywhere logic may be confusing without context

This fixes LAND-1521 removing Talk and all associated code. It also removes the wayfinding button and moves the privacy notice and help and support options into the main app menu.

### CODE REVIEW

**Comment by arthyn** [comment]
Targeting Brian's branch, but this should be merged to develop only after his branch goes to staging. This will ensure that people can get the deprecation message deployed to Talk first and then it will receive no other deploys.

File: ui/src/dms/MobileMessagesSidebar.tsx [hunk 1]
```diff
@@ -2,100 +2,32 @@ import { useRef, useState } from 'react';
...
-  const { mutate } = usePutEntryMutation({
-    bucket: 'talk',
-    key: 'messagesFilter',
-  });
-  const pinned = usePinnedChats(appName === 'Groups');
-
-  const setFilterMode = (mode: SidebarFilter) => {
-    mutate({ val: mode });
-  };
+  // TODO: may need to tweak
```
Comment (latter-bolden) [inline ui/src/dms/MobileMessagesSidebar.tsx:17]: I think this is what we want here, messages tab on mobile shouldn't have chats

File: desk/app/chat.hoon [hunk 2]
```diff
@@ -1089,18 +1089,11 @@
...
-    |=  [con=(list content:ha) but=(unit button:ha)]
-    ^-  new-yarn:ha
-    =/  rope  [~ ~ %groups /club/(scot %uv id)]
-    =/  link  /dm/(scot %uv id)
-    [& & rope con link but]
-  ::
   ++  cu-spin
     |=  [rest=path con=(list content:ha) but=(unit button:ha)]
     ^-  new-yarn:ha
     ::  hard coded desk because these shouldn't appear in groups
```
Comment (latter-bolden) [inline desk/app/chat.hoon:1095]: should remove outdated comment?

File: desk/app/chat.hoon [hunk 3]
```diff
@@ -1504,18 +1488,11 @@
...
-    |=  [con=(list content:ha) but=(unit button:ha)]
-    ^-  new-yarn:ha
-    =/  rope  [~ ~ %groups /dm/(scot %p ship)]
-    =/  link  /dm/(scot %p ship)
-    [& & rope con link but]
-  ::
   ++  di-spin
     |=  [rest=path con=(list content:ha) but=(unit button:ha)]
     ^-  new-yarn:ha
     ::  hard coded desk because these shouldn't appear in groups
```
Comment (latter-bolden) [inline desk/app/chat.hoon:1494]: same here?

**Review by latter-bolden — APPROVED** [review]
Might want another set of eyes on the hoon changes, but this is looking good to me sans-conflicts. Couple of very minor notes

File: ui/src/components/Settings/Settings.tsx [hunk 4]
```diff
@@ -128,25 +110,23 @@ export default function Settings() {
...
-              Groups experience.
-            </p>
-          </Setting>
-        )}
-        {isGroups && logActivity && (
+        <Setting
+          on={logActivity}
+          toggle={() => toggleLogActivity({ val: !logActivity })}
+          status={logActivityStatus}
+          name="Log Groups Usage"
```
Comment (patosullivan) [inline ui/src/components/Settings/Settings.tsx:120]:
```suggestion
          name="Log Usage"
```

File: ui/src/components/Settings/Settings.tsx [hunk 5]
```diff
@@ -128,25 +110,23 @@ export default function Settings() {
...
+          labelClassName="font-semibold"
+        >
+          <p className="leading-5 text-gray-600">
+            Enable or disable basic activity tracking in Groups. Tlon uses this
```
Comment (patosullivan) [inline ui/src/components/Settings/Settings.tsx:124]:
```suggestion
            Enable or disable basic activity tracking. Tlon uses this
```

File: ui/src/components/Settings/Settings.tsx [hunk 6]
```diff
@@ -128,25 +110,23 @@ export default function Settings() {
...
+            data to make product decisions and to bring you a better Groups
+            experience.
```
Comment (patosullivan) [inline ui/src/components/Settings/Settings.tsx:126]:
```suggestion
            data to make product decisions and to bring you a better
            experience.
```

File: ui/src/logic/analytics.ts [hunk 7]
```diff
@@ -80,8 +80,8 @@ export const captureAnalyticsEvent = (
   name: AnalyticsEventName,
   properties?: Properties
 ) => {
-  // Do not capture any analytics events for Talk
-  if (isTalk || isNativeApp()) {
+  // TODO: verify we actually want this
+  if (isNativeApp()) {
```
Comment (patosullivan) [inline ui/src/logic/analytics.ts:84]: Seems like we shouldn't want this imo

File: ui/src/state/settings.ts [hunk 8]
```diff
@@ -350,8 +347,8 @@ export function useLogActivity() {
   const { data, isLoading } = useMergedSettings();
 
   return useMemo(() => {
-    // Do not capture any analytics events for Talk
-    if (isTalk || isNativeApp()) {
+    // TODO: is this correct?
```
Comment (patosullivan) [inline ui/src/state/settings.ts:350]: Again, feels like no

**Review by patosullivan — APPROVED** [review]
Looks good! Some minor changes to copy and I feel like we ought to not no-op on analytics for the native app, that doesn't make much sense.

**Pull request merged**